packages/react/package.json

```diff
@@ -54,37 +54,37 @@
     "coverage:codecov": "yarn test --coverage && codecov"
   },
   "dependencies": {
-    "@babel/runtime": "^7.6.2",
-    "@react-aria/button": "3.6.0",
-    "@react-aria/checkbox": "3.5.0",
-    "@react-aria/dialog": "3.3.0",
-    "@react-aria/focus": "3.7.0",
-    "@react-aria/i18n": "3.5.1",
-    "@react-aria/interactions": "3.10.0",
-    "@react-aria/label": "3.4.0",
-    "@react-aria/link": "3.3.1",
-    "@react-aria/menu": "3.6.0",
-    "@react-aria/overlays": "3.10.0",
-    "@react-aria/radio": "3.3.0",
+    "@babel/runtime": "^7.20.1",
+    "@react-aria/button": "3.6.2",
+    "@react-aria/checkbox": "3.6.0",
+    "@react-aria/dialog": "3.4.0",
+    "@react-aria/focus": "3.9.0",
```

Comment:
From [`v3.8.0`](https://github.com/adobe/react-spectrum/issues/3458) it will break the app and it could be useful to specify in the docs that strict-mode should be disabled.